src/components/Facepile/Facepile.tsx

```diff
@@ -35,19 +37,19 @@ export class Facepile extends React.Component<IFacepileProps, {}> {
             })
           }
           { this.props.overflowButtonProps ? this._getOverflowElement() : null }
-          { this.props.chevronButtonProps && personalDetailsHidden ? this._getChevronElement() : null }
+          { this.props.chevronButtonProps && !this._personaDetailsShown ? this._getChevronElement() : null }
         </div>
       </div>
     );
   }
 
   private _getPersonasToDisplay(): IFacepilePersona[] {
-    let personasToShow: IFacepilePersona[] = this.props.personas.slice(0, this.numPersonasToShow);
+    let personasToShow: IFacepilePersona[] = this.props.personas.slice(0, this._numPersonasToShow);
     return personasToShow;
   }
 
   private _getPersonasNotToDisplay(): IFacepilePersona[] {
-    let personasToShow: IFacepilePersona[] = this.props.personas.slice(this.numPersonasToShow);
```

Comment:
_getPersonasNotToDisplay isn't necessary. Could just be generated in `render` or wherever it is used. 

```
const tooltipText = personas.slice(visibleCount).map(p => p.name).join(', ');
```